### PR TITLE
Fix tuning set for the ILB test

### PR DESCRIPTION
**Pull request by tosi3k**
`qpsLoad` does not have a `averageQps` field - this should be renamed to `qps`.

Also lowering the QPS to 20.

### CODE REVIEW

File: clusterloader2/testing/l4ilb/config.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 {{$MEDIUM_BACKEND_LB_SERVICE_COUNT := DefaultParam .CL2_MEDIUM_BACKEND_LB_SERVICE_COUNT 2}}
 {{$SMALL_BACKEND_LB_SERVICE_COUNT := DefaultParam .CL2_SMALL_BACKEND_LB_SERVICE_COUNT 2}}
 # adding a fixed value for first version of the test, rate of pod creation not a concern yet.
-{{$ilbQPS := 500}}
+{{$ilbQPS := 20}}
```
Comment (jkaniuk) [inline clusterloader2/testing/l4ilb/config.yaml:12]: Could you make it customizable as well?

Comment (tosi3k) [reply clusterloader2/testing/l4ilb/config.yaml]: +1. Done.

**Comment by jkaniuk** [comment]
/lgtm
/approve

@prameshj

**Pull request merged**